src/app/components/result/result.component.ts

```diff
@@ -32,7 +32,26 @@ export class ResultComponent implements OnInit, OnDestroy {
     'warning': 'fa-exclamation-triangle',
     'error': 'fa-times-circle',
     'critical': 'fa-times-circle'
-  }
+  };
+  public severityLevelNames = {
+    'info': $localize `Info`,
+    'notice': $localize `Notice`,
+    'warning': $localize `Warning`,
+    'error': $localize `Error`,
+    'critical': $localize `Critical`,
+  };
+  public moduleNames = {
+    'system': $localize `System`,
+    'basic': $localize `Basic`,
+    'address': $localize `Address`,
+    'connectivity': $localize `Connectivity`,
+    'consistency': $localize `Consistency`,
+    'delegation': $localize `Delegation`,
+    'dnssec': $localize `DNSSEC`,
+    'nameserver': $localize `Nameserver`,
+    'syntax': $localize `Syntax`,
+    'zone': $localize `Zone`,
```

Comment:
This is a list of the current modules. What is a new module name comes, e.g. the name of a custom module?

Reply:
That’s a limitation of the current design. Shouldn’t these module names be moved over to Zonemaster::Engine, eventually?

Reply:
If custom module names just show up untranslated, it is fine. If they disappear, it is not fine

I think we should consider moving it to Engine later but it is fine to have it in GUI for now.